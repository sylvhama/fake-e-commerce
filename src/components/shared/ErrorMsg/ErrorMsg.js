import styled from 'styled-components';

export default styled.p`
  padding: ${props => props.theme.spacing.reg};
  color: ${props => props.theme.color.primary};
  text-align: center;
`;
