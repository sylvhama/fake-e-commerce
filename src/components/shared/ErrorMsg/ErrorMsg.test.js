import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../tests/helpers';

import ErrorMsg from './ErrorMsg';

test('ErrorMsg renders correctly', () => {
  const rendered = renderWithTheme(<ErrorMsg>ErrorMsg</ErrorMsg>).toJSON();
  expect(rendered).toMatchSnapshot();
});
