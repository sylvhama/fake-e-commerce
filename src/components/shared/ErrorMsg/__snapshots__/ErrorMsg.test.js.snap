// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMsg renders correctly 1`] = `
.c0 {
  padding: 16px;
  color: #ea352d;
  text-align: center;
}

<p
  className="c0"
>
  ErrorMsg
</p>
`;
